docs/application/dotnet/guides/app-management/application-launcher.md

```diff
@@ -0,0 +1,304 @@
+# Application launcher
+
+This guide explains how to create a basic application launcher. The application launcher is a main application that normally starts after system boot. This app is commonly replaced in the platform adjustments process. It is easy to use the .NET APIs in the application launcher implementation.
```

Comment:
Did not understand this sentence could you help understand:
This app is commonly replaced in the platform adjustments process.

```suggestion
This guide explains how to create a basic application launcher. The application launcher is a main application that normally starts after system boot. This app is commonly replaced in the platform adjustment process. It is easy to use the .NET APIs in the application launcher implementation.
```

Reply:
When Tizen platform will be used by other developer in their platform, homescreen app is ofter replaced. I want to wirte something like this. Please let me know if my explanation is clear.